### PR TITLE
Update office-365-network-mac-perf-onboarding-tool.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note stating that it is important to sign in to the tenant before running the test as it uploads and shares the details with the admin.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3356

File: microsoft-365/enterprise/office-365-network-mac-perf-onboarding-tool.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ description: "Microsoft 365 network connectivity test tool (preview)"
 
 The Microsoft 365 network connectivity test tool is located at <https://connectivity.office.com>. It is an adjunct tool to the network assessment and network insights information available in the Microsoft 365 admin center under the **Health | Connectivity** menu.
 
+>[!IMPORTANT]
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/office-365-network-mac-perf-onboarding-tool.md]:
```suggestion
> [!IMPORTANT]
```

File: microsoft-365/enterprise/office-365-network-mac-perf-onboarding-tool.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ description: "Microsoft 365 network connectivity test tool (preview)"
...
+> It is important to sign in to Your M365 tenant as All test reports while you are signed in are shared with your administrator and uploaded to the tenant.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/office-365-network-mac-perf-onboarding-tool.md]:
```suggestion
> It is important to sign in to your Microsoft 365 tenant as all test reports are shared with your administrator and uploaded to the tenant while you are signed in.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks.

**Pull request merged**